### PR TITLE
Pull request for install-curl-before-using-it

### PR DESCRIPTION
**Pull request by fblackburn1**
install curl before using it
fix styling issues
set "p" option by default

### CODE REVIEW

File: wazo_install.sh [hunk 1]
```diff
@@ -122,15 +122,22 @@ install_wazo_with_ansible() {
     apt-get purge --autoremove -yq python3-pip virtualenv
 }
 
+install_prerequisite() {
+    if [ $ansible_tag == 'LAST-VERSION' ]; then
+        apt-get update
+        apt-get install -yq curl
+        ansible_tag=wazo-$(curl https://mirror.wazo.community/version/stable)
```
Comment (sduthil) [inline wazo_install.sh]: Please don't change the variable `ansible_tag` in the function called `install_prerequisite`

Comment (fblackburn1) [reply wazo_install.sh]: Ok :ok_hand: 
I renamed the function, but have you a better approach to fix it?
This PR is more an idea than a lovely solution :confused:

**Comment by fblackburn1** [comment]
This PR fix https://jenkins.wazo.community/job/daily-acceptance-ha/

**Comment by sduthil** [comment]
recheck

**Pull request merged**